src/pages/fr/guides/rss.mdx

```diff
@@ -53,18 +69,19 @@ export const get = () => rss({
 
 Les `items` acceptent soit :
 
-1. [Un résultat de `import.meta.glob(...)`](#1-avec-un-résultat-importmetaglob) **(utilisez cette option pour les fichiers `.md` à l'interieur du répertoire `src/pages/`)**
+1. [Un résultat de `import.meta.glob(...)`](#1-avec-un-résultat-importmetaglob) **(utilisez cette option uniquement que pour les fichiers `.md` à l'intérieur du répertoire `src/pages/`)**
```

Comment:
Hi,
The "que" in the string "uniquement que pour les fichiers" should be removed. 

It looks like a mix of the different ways to say the same thing, thus not a real expression.